draft-irtf-t2trg-rest-iot.md

```diff
@@ -541,7 +541,7 @@ The PATCH method is not safe nor idempotent.
 
 The CoAP-specific iPATCH method is a variant of the PATCH method that is not safe, but is idempotent.
 
-## HTTP/CoAP Status/Response Codes
+## HTTP/CoAP Response Codes
 
 {{Section 6 of RFC7231}} defines a set of Status Codes in HTTP that are assigned by the server to indicate whether a request was understood and satisfied, and how to interpret the answer.
```

Comment:
7231 is obsoleted by 9110/9112

Reply:
Seems [Section 15 of RFC9110](https://datatracker.ietf.org/doc/html/rfc9110#name-status-codes) is then the right reference here.

Reply:
Actually this was already fixed in the [main branch](https://github.com/t2trg/t2trg-rest-iot/blob/master/draft-irtf-t2trg-rest-iot.md?plain=1#L552).